### DISCUSSION TITLE
Maximum call stack size exceeded with ShadowDOM

### DISCUSSION THREAD

**Discussion by dakom** [floating-ui/floating-ui#1288] [Q&A]
My build is quite complicated, mixture of Rust/WASM and Lit-Element and Popper.js

If I isolate this issue to a smaller reproducible codesandbox or temp repo, I'll update the issue here.

In the meantime, I am getting `Maximum call stack size exceeded` errors in `getWindow` and `getComputedStyle`

Strangely, this happens only on remote deployment, not in my local build and I'm having a tough time reproducing it. Maybe it's a bundling/rollup issue, but not sure.

Sorry for the vague info at this point - though maybe someone else hit this, and I do believe the "weirdest" thing in my setup is passing element references across different shadow dom trees.

**Comment by nothingrandom** [comment]
Did you ever find a solution to this?

Having this on some developer machines locally and not other machines, which is really odd.

**Reply by dakom** [reply]
I ended up writing a proprietary anchoring system... sorry!
